### DISCUSSION TITLE
Welcome to egg Discussions!

### DISCUSSION THREAD

**Discussion by melkyades** [powerlang/egg#15] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by KenDickey** [comment]
Hobby Programmer interested in self hosting and compile-to-native.  I want to get rid of complex and stupid intermediate tools and languages.
Old runtime guy.  Ported 68k Scheme compiler & runtime to Amiga.  Was at Tektronix doing a Scheme compiler & runtime for Moto 88K and chatting with the Instantiations Magnolia/68k Smalltalk team; at Apple doing SK8 Multimedia IDE during Squeak development. Knew folks at ParkPlace Systems. Looked at Dave Simmons' Smalltalk runtime for Apple.  Done Scheme/Lisp commercial projects.  Basically a runtime guy.  Last couple of decades playing in Cuis Smalltalk.  Did initial port of OpenSmalltalk-VM stack interpreter to aarch64/arm64 and riscv64.  Various random stuff.
So a bunch of "guilt by association".
